x-pack/plugins/lists/common/schemas/common/schemas.ts

```diff
@@ -130,6 +130,10 @@ export type NamespaceType = t.TypeOf<typeof namespace_type>;
 
 export const operator = t.keyof({ excluded: null, included: null });
 export type Operator = t.TypeOf<typeof operator>;
+export enum OperatorEnum {
```

Comment:
Icky, I know. It's a workaround to enums not being a thing in io-ts yet - https://github.com/gcanti/io-ts/issues/67 . I've added unit tests so that if the io-ts type `operator` is updated, tests will fail and user will see note about needing to update `OperatorEnum` as well.